sample/Sample/Sample.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net7.0;net462</TargetFrameworks>
+    <TargetFrameworks>net6.0;net7.0;net462;net8.0</TargetFrameworks>
```

Comment:
nit: `net6.0;net7.0;net8.0;net462` would be easier to the eyes than `net6.0;net7.0;net462;net8.0`